components/PreviewFrame/index.js

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState, useRef } from 'react'
+import { useColorContext } from '@project-r/styleguide'
+import { FRONTEND_BASE_URL } from '../../lib/settings'
+
+import { SIDEBAR_WIDTH } from '../Sidebar'
+
+const PREVIEW_PADDING = 16
```

Comment:
```suggestion
const PREVIEW_MARGIN = 16
```
No biggie, but this got me confused now that I'm having a look to the centering issue.

Reply:
done